services/static-webserver/client/source/class/osparc/store/Store.js

```diff
@@ -618,7 +618,7 @@ qx.Class.define("osparc.store.Store", {
     __getOrgClassifiers: function(orgId, useCache = false) {
       const params = {
         url: {
-          "gid": orgId
+          "gid": parseInt(orgId)
```

Comment:
You really get this as `str`?  Is there some field that I miss in the API? 
I guess if you get an `int`, `parseInt` does not fail